### DISCUSSION TITLE
23.02 OIDC / LDAP 500 error

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1422] [Q&A] [answered]
I am running 23.02  . Referencing your openDCIM OIDC Authentication with AzureAD. The  the OIDC/LDAP is missing.  Actually the configuration page seems to have not changed at all .   Can you share what needs to be in place to make this work .    I have a registered application ready to configure .

**Comment by mfenton01** [comment]
I get the 500 error when I change authentication to OIDC thinking that may bring up the changes needed .

**Comment by samilliken** [comment]
You are likely missing a dependency package.   You need to post details about your environment (LAMP or container, OS and version).   Check the apache error logs - they will most likely state that you are trying to call a function that doesn't exist, which will give a clue as to what is missing.

**Reply by mfenton01** [reply]
VM 
RHEL 8.7
Apache/2.4.37 (Red Hat Enterprise Linux) 
PHP Version 8.0.27
MySQL Ver 15.1 Distrib 10.3.35-MariaDB

Nothing unusual in error log 

No app errors since initial install  in var/log/httpd/error.log .  
error basically couldnt find the drawings .. I always fix that last anyway.

**Reply by wilpig** [reply, marked as answer]
This should be all the required packages.  https://github.com/opendcim/openDCIM/wiki/RHEL#install-all-the-components-that-are-necessary

**Reply by mfenton01** [reply]
This helped a lot .. Thank you 
![2023-08-17_openDCIM 23 02](https://github.com/opendcim/openDCIM/assets/2854433/ef154d44-4787-42cb-8dab-b562415d1c30)

**Reply by mfenton01** [reply]
After following the PHP updates / Adds all went green .
